i18n/en.json

```diff
@@ -17,6 +17,7 @@
 	"createwiki-error-notalnum": "The database name may not contain non-alphanumeric characters.",
 	"createwiki-error-notlowercase": "The database name may not contain uppercase letters.",
 	"createwiki-error-notsuffixed": "The database name doesn't end in a valid suffix (e.g. 'wiki').",
+	"createwiki-error-swiftcontainernot": "Failed to create swift container for wiki $1",
```

Comment:
why $1? you don't pass a variable and its fairly obvious which wiki it is to the wiki creator.

Reply:
Now you pass a variable.